gui/hud.gui

```diff
@@ -2994,6 +2994,181 @@ widget = {
 		}
 
 		widget_stress_sounds = {}
+
+		#Warcraft
+		spacer = {
+			size = { 9 0 }
+		}
+
+		infusion_bar = {
+			#Bar
+			blockoverride "stretch_1"
+			{
+				layoutstretchfactor_horizontal = "[GetProgressBarValueMaxScaled( FixedPointToFloat(GuiScope.SetRoot( GetPlayer.MakeScope ).ScriptValue('light_void_bar_stretch_value')), '(float)200', '(int32)80' )]"
+			}
+			blockoverride "stretch_2"
+			{
+				layoutstretchfactor_horizontal = "[GetProgressBarValueMaxScaled( FixedPointToFloat(GuiScope.SetRoot( GetPlayer.MakeScope ).ScriptValue('light_void_bar_stretch_value')), '(float)200', '(int32)80' )]"
+			}
```

Comment:
I wonder if there can be 2 blocks of the same name so we make one blockoverride since I see these 2 are identical.

Reply:
No, tried it before, didn't work.